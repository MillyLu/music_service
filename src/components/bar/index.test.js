/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import Bar from './bar';



describe('<Bar />', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<Bar />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });
