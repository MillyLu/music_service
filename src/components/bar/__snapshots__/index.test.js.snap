// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bar /> should render successfully and match snapshot 1`] = `
<styled.div
  theme={0}
>
  <BarContent />
</styled.div>
`;
